man/docker-tag.1.md

```diff
@@ -34,7 +34,8 @@ separated by a ':'
    The tag you are assigning to the image.  Though this is arbitrary it is
 recommended to be used for a version to distinguish images with the same name.
 Also, for consistency tags should only include a-z0-9-_. .
-Note that here TAG is a part of the overall name or "tag".
+Note that here TAG is a part of the overall name or "tag". Refer to 
+**docker-tag(1)** for more information about valid image and tag names.
```

Comment:
This is the "docker-tag(1)" man-page, so we should copy the information here that's in the docs/reference/commandline/tag.md document.